app/controllers/case_contacts/form_controller.rb

```diff
@@ -25,13 +25,7 @@ def update
     if @case_contact.active?
       # do nothing
     else
-      begin
-        params[:case_contact] ||= []
-        params[:case_contact][:status] = step.to_s # TODO: where is this used?? what is it for??
-      rescue => e
-        # TODO https://app.bugsnag.com/ruby-for-good/casa/errors/6637007c6857010008cfc9dd
-        Bugsnag.notify(e)
-      end
+      params[:case_contact][:status] = step.to_s
     end
```

Comment:
`params[:case_contact][:status] = step.to_s unless @case_contact.active?`

is what it was in the past unless you think this is more clear.

Reply:
I like it in the long form tbh